src/app/docs/autonumeric/autonumeric-docs.component.html

```diff
@@ -4,22 +4,32 @@
   pageTitle="Autonumeric"
 >
   <sky-docs-demo-page-summary>
-    The autonumeric module automatically formats currency and other numbers that users enter in form inputs.
+    The autonumeric directive automatically formats currency and other numbers that users enter in form inputs. This directive wraps the third-party autoNumeric library. If you have never used autoNumeric, <a href="https://github.com/autoNumeric/autoNumeric#what-is-autonumeric">the "What is autoNumeric?"</a> guide covers the basics. For more in-depth information about autoNumeric and its capabilities, see <a href="https://github.com/autoNumeric/autoNumeric/blob/next/README.md">the autoNumeric documentation.</a>
   </sky-docs-demo-page-summary>
 
-<!--
-  <sky-docs-demo>
-    See documentation for how to setup demos
-  </sky-docs-demo>
--->
-<!--
   <sky-docs-code-examples
     [packageDependencies]="{
-      '@skyux/autonumeric': '*'
+      '@skyux/autonumeric': '*',
+      'autonumeric': '*'
     }"
   >
-    See documentation for how to setup code examples.
+    <sky-docs-code-example
+      heading="Basic"
```

Comment:
```suggestion
      heading="Modified decimal placement and character"
```
Is "Basic" an accurate label here? I think the "Basic" code samples are usually the most vanilla version of whatever we are demonstrating. But in this case, we're demonstrating some atypical behaviors. I'm NOT confident that my suggested heading is better or that this matters much either way. But just throwing something out there to see what you think. Maybe just "Modified decimal" would work too?

Reply:
Hmmm. That one is tough. That title seems a little too specific - how about we title this one "International formatting"? I think that better describes why you would be doing the thing that the code example is showing.

Reply:
Is this actual international formatting? I thought that when we talked about it yesterday, this was an exaggerated version of international formatting. If I got that wrong, then I think "International formatting" works great. But if not, would "Modified formatting" work instead?

Reply:
I think I prefer International Formatting. 

The 4 decimal places might not be "true" international formatting but in my opinion it is close enough for that heading to be accurate and best describes what users will be using this for. I want to leave the 4 decimals as it shows users how to use that even if it isn't 100% accurate. And though our users do tend to copy and paste I believe this is an example where they will have to change that anyway for their needs and I'm not worried about that.

Reply:
I've updated this to "International formatting"

Reply:
Sounds good. Thanks!